acceptance/auth/bundle_and_profile/script.cleanup

```diff
@@ -0,0 +1 @@
+rm -f ~/.databrickscfg
```

Comment:
You don't need to use script.prepare (and script.cleanup) within context of single script. They will be just concatenated into one. This feature is meant for cases where prepare and cleanup are shared.